who-metrics-ui/src/components/RepositoriesTable.tsx

```diff
@@ -455,6 +475,53 @@ const RepositoriesTable = () => {
         </a>
       ),
     },
+    Topics: {
+      key: 'topics',
+      name: 'Topics',
+      width: 275,
+      renderHeaderCell: (p) => {
+        return (
+          <SearchableSelectRenderer
+            headerCellProps={p}
+            filterName="topics"
+            filters={globalFilters}
+            updateFilters={setGlobalFilters}
+          />
+        )
+      },
+      renderCell: (props) => {
+        const [isHovering, setIsHovering] = useState(false)
+
+        // tabIndex === 0 is used as a proxy when the Cell is selected. See https://github.com/adazzle/react-data-grid/pull/3236
+        const isOpen = props.row.topics.length > 0 && (isHovering || props.tabIndex === 0)
+
+        return (
+          <Popover isOpen={isOpen} content={() => {
+            return (
+              <Box
+                className="shadow-xl min-w-64 p-4 rounded"
+                onClick={(e) => e.stopPropagation()}
+                sx={{
+                  backgroundColor: 'Background',
+                  border: '1px solid',
+                  borderColor: 'border.default',
+                }}
+              >
+                {props.row.topics.sort().join(", ")}
```

Comment:
Whatcha think about doing the topic badge like in github 🤔 I think the API returns the color of it

Reply:
Oh wait, I'm thinking of labels. Topics are just a single color, whoops!

Reply:
It still might be nicer to render these as labels instead, we can use the default variant. Will update!  https://primer.style/react/Label

Reply:
Updated in https://github.com/sbv-world-health-org-metrics/sbv-world-health-org-metrics/pull/100/commits/9f949e80fdcfeee14b93eecbbf3277e1bfa85e93